addons/account/views/partner_view.xml

```diff
@@ -158,14 +158,14 @@
                 </div>
 
                 <page name="internal_notes" position="inside">
-                    <t groups="account.group_account_invoice,account.group_account_readonly">
-                        <group colspan="2" col="2" groups="account.group_warning_account">
-                            <separator string="Warning on the Invoice" colspan="4"/>
-                            <field name="invoice_warn" nolabel="1" />
-                            <field name="invoice_warn_msg" colspan="3" nolabel="1"
+                    <group groups="account.group_account_invoice,account.group_account_readonly">
+                        <group groups="account.group_warning_account" col="2">
+                            <separator string="Warning on the Invoice" colspan="2"/>
+                            <field name="invoice_warn" nolabel="1" colspan="2"/>
```

Comment:
Because there are no labels and no borders for the fields, we need something here
```suggestion
                            <field name="invoice_warn" nolabel="1" colspan="2" required="1"/>
```

Reply:
hi @william-andre ,  updated the pr as per the suggestion